www/templates/html/MediaPlayer.tpl.php

```diff
@@ -4,15 +4,175 @@
 } else {
     echo $savvy->render($context->media, 'MediaPlayer/Audio.tpl.php');
 }
+
+$jsonTrack = @file_get_contents($context->media->getVideoTextTrackURL("json")); 
```

Comment:
Is this variable really needed anymore? (it's used in various places below still)